test/gui/shared/scripts/names.py

```diff
@@ -107,3 +107,13 @@
 oCC_SslErrorDialog_cbTrustConnect_QCheckBox = {"name": "_cbTrustConnect", "type": "QCheckBox", "visible": 1, "window": oCC_SslErrorDialog_OCC_SslErrorDialog}
 oCC_SslErrorDialog_OK_QPushButton = {"text": "OK", "type": "QPushButton", "unnamed": 1, "visible": 1, "window": oCC_SslErrorDialog_OCC_SslErrorDialog}
 owncloudWizard_urlLabel_QLabel = {"name": "urlLabel", "type": "QLabel", "visible": 1, "window": owncloudWizard_OCC_OwncloudWizard}
+bSelectiveSync_QPushButton = {"name": "bSelectiveSync", "type": "QPushButton", "visible": 1, "window": owncloudWizard_OCC_OwncloudWizard}
```

Comment:
this selector is also being used in only one page object. So, may be you can remove it from names.py and keep it in the page object itself.